gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/drilling/Drill.java

```diff
@@ -232,7 +316,8 @@ public DrillUp buildDrillUp() {
     public DrillDown buildDrillDown() {
       Objects.requireNonNull(propertyKey);
       return new DrillDown(
-        label, propertyKey, function, newPropertyKey, drillVertex);
+        label, propertyKey, vertexDrillFunction, edgeDrillFunction, graphheadDrillFunction,
```

Comment:
IllegalArgumentException (in case of no functions given) should be thrown here as well.

Reply:
A DrillDown without a specific DrillFunction is valid in case of a preceded DrillUp. By default it is then applied to vertices.